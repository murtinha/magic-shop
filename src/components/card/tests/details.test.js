import React from 'react';
import { shallow } from 'enzyme';
import CardDetails from '../details';

describe('<CardDetails />', () => {
  const props = {
    image: 'path/imagem.png',
    name: 'Black Lotus',
    manaCost: '0',
    type: 'Artifact',
    rarity: 'Rare',
    setName: 'Magic collection',
    text: 'Card text',
    flavor: 'Card flavor',
    artist: 'murtinha',
  };

  describe('render', () => {
    it('should render properly', () => {
      const wrapper = shallow(<CardDetails {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
