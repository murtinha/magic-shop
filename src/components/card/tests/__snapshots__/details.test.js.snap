// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardDetails /> render should render properly 1`] = `
<Portal>
  <div
    className="wrap"
  >
    <img
      alt="imagem"
      className="image"
      src="path/imagem.png"
    />
    <div
      className="detailsWrapper"
    >
      <div
        className="text"
      >
         Nome: 
        Black Lotus
         
      </div>
      <div
        className="text"
      >
         Custo de mana: 
        0
         
      </div>
      <div
        className="text"
      >
         Tipo: 
        Artifact
         
      </div>
      <div
        className="text"
      >
         Raridade: 
        Rare
         
      </div>
      <div
        className="text"
      >
         Coleção: 
        Magic collection
         
      </div>
      <div
        className="text"
      >
         Texto: 
        Card text
         
      </div>
      <div
        className="text"
      >
         Descrição: 
        Card flavor
         
      </div>
      <div
        className="text"
      >
         Artista: 
        murtinha
         
      </div>
    </div>
  </div>
</Portal>
`;
